src/demo-app/expansion/expansion-demo.html

```diff
@@ -0,0 +1,58 @@
+<h1>Single Expansion Panel</h1>
+<md-expansion-panel class="md-expansion-demo-width" #myPanel>
+  <md-expansion-panel-header>
+    <span class="md-expansion-panel-header-description">This is a panel description.</span>
+    <span class="md-expansion-panel-header-title">Panel Title</span>
```

Comment:
Sorry, was still unclear. I had meant last time (plus shortening the names somewhat):
```html
<md-expansion-panel class="md-expansion-demo-width" #myPanel>
  <md-panel-header>
    <md-panel-description> ... </>
    <md-panel-title> ... </>
  </>
</>
```